docs/cloud/private_connectivity_sourcegraph_connect.mdx

```diff
@@ -1,115 +1,115 @@
+Tunnel agent: deployed inside the customer's network, has its own identity, proxies and encrypts traffic between the code host and tunnel client. The agent can only communicate with permitted private resources inside the customer's network. The customer starts the tunnel agent with the credentials provided by Sourcegraph, then the agent authenticates and establishes a secure connection with the tunnel server. Only agents are allowed to establish secure connections with the tunnel server, and the server only accepts a connection if the agent's identity is approved.
 
-Tunnel server: a centralized broker between client and agent managed by Sourcegraph. Its purpose is to set up mTLS, proxy encrypted traffic between clients and agents and enforce ACL.
-
-Tunnel client: forward proxy clients managed by sourcegraph. Every client has its own identity and it cannot establish a direct connection with the customer agent, and has to go through tunnel server.
-
-[link](https://link.excalidraw.com/readonly/453uvY8infI8wskSecGJ)
+Tunnel client: forward proxy clients managed by Sourcegraph, added to the customer's Sourcegraph Cloud instance. Every client has its own identity and it cannot establish a direct connection with the customer agent, and has to go through tunnel server.
 
 <iframe
   src="https://link.excalidraw.com/readonly/453uvY8infI8wskSecGJ"
   width="100%"
   height="100%"
   style={{ border: "none" }}
 />
+[Diagram link](https://link.excalidraw.com/readonly/453uvY8infI8wskSecGJ)
 
 ## Steps
 
 ### Initiate the process
 
-Customer should reach out to their account manager to initiate the process. The account manager will work with the customer to collect the required information and initiate the process, including but not limited to:
+The customer reaches out to their account manager to initiate the process. The account manager will work with the customer to collect the required information and initiate the process, including but not limited to:
 
-- The DNS name of the private code host, e.g. `gitlab.internal.company.net` or private artifact registry, e.g. `artifactory.internal.company.net`.
-- The port of the private code host, e.g., `443`, `80`, `22`.
-- The type of TLS certificate used by the private resource: either self-signed by an internal private CA or issued by a public CA.
+- The DNS name of the private code host, e.g. `gitlab.internal.company.net` or private artifact registry, e.g. `artifactory.internal.company.net`
+- The port of the private code host, e.g., `443`, `80`, `22`
+- The type of TLS certificate used by the private resource: self-signed, internal PKI, or issued by a public CA
```

Comment:
```suggestion
- The type of TLS certificate used by the private resource: self-signed by an internal PKI, or issued by a public CA
```

Reply:
Suggestion noted. Self-signed and internal PKI are two separate categories of TLS certificates, and are handled differently when we configure the certs in their instance's site config. A code host using a self-signed cert means it wasn't signed by a CA, so we have to add that cert's public key to the site config. A code host using a cert signed by the customer's internal PKI means we can configure their root CA's public key in the site config, and all certs signed by any intermediate CA signed by the same root CA will be trusted, so we often only need to manage one cert instead of many.